docs/getting-started/create-pipeline.md

```diff
@@ -0,0 +1,61 @@
+# Creating a Pipeline
+Using the Screwdriver CLI, you can build pipelines locally on your machine.
+
+## Set up Screwdriver CLI
+
+**The Screwdriver CLI does not exist yet, but here is what the instructions would looks like.**
+
+### Installing the CLI
+
+```bash
+$ go get github.com/screwdriver-cd/client
+$ $GOPATH/bin/client
```

Comment:
Yeah, I don't think we should be installing Go, we should be installing the client.  Example: 

```bash
$ wget https://github.com/screwdriver-cd/client/releases/download/v1.0.2/sd
$ ./sd repl
```